Makefile

```diff
@@ -19,18 +20,29 @@ JS_TESTING_FILES := $(wildcard test/test*.js test-whitelist/test*.js)
 NODE ?= node
 NPM ?= $(NODE) $(shell which npm)
 MOCHA ?= $(NODE) $(BIN)/mocha
+BABEL ?= $(NODE) $(BIN)/babel
 WEBPACK ?= $(NODE) $(BIN)/webpack
 
-standalone: build/wpcom.js
+build: mkbuilddir
```

Comment:
The main problem here is that you're not depending on the source files directly (to the right hand side of the `:`), so `make build` will _always_ run babel, even when no changes have been made. This defeats the point of using Make in the first place.